packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx

```diff
@@ -78,6 +80,25 @@ switch (literal) {
 }
 ```
 
+### defaultCaseCommentPattern
+
+{/* insert option description */}
+
+Basically, there is logic to treat annotations matching /^no default$/iu as default case.
+If you want to change this pattern, just enter the pattern in this option.
```

Comment:
[Docs] Cleanup for clarity:

```suggestion
It can sometimes be preferable to omit the default case for only some switch statements.
For those situations, this rule can be given a pattern for a comment that's allowed to take the place of a `default:`.
```

https://jameshfisher.com/2017/02/22/dont-use-simply - same with _"just"_.

As of #9925, the option's default value is explicitly stated in the automatically added description.

Reply:
The above link helped me a lot. Thank you!